.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: node_js
 node_js:
-    - "6"
```

Comment:
I'd love to keep 6 and 8, and add 9 instead, so we have coverage for all (relevant) LTS versions and for the current node version.